src/marine/ndbc_hfradar2ioda.py

```diff
@@ -120,18 +118,18 @@ def main():
         'sea_water_meridional_current': ['nlocs'],
         'sea_water_zonal_current': ['nlocs']}
     radar = Observation(args.input, fdate)
-    GlobalAttrs['date_time_string'] = fdate.strftime("%Y-%m-%dT%H:%M:%SZ")
 
     ObsVars, nlocs = iconv.ExtractObsData(radar.data, locKeyList)
-    DimDict = {'nlocs': nlocs}
+    DimDict = {'Location': nlocs}
     writer = iconv.IodaWriter(args.output, locKeyList, DimDict)
     VarAttrs = DefaultOrderedDict(lambda: DefaultOrderedDict(dict))
-    VarAttrs[('sea_water_meridional_current', 'ObsValue')]['units'] = 'm/s'
-    VarAttrs[('sea_water_zonal_current', 'ObsValue')]['units'] = 'm/s'
-    VarAttrs[('sea_water_meridional_current', 'ObsValue')]['_FillValue'] = -32767
-    VarAttrs[('sea_water_zonal_current', 'ObsValue')]['_FillValue'] = -32767
-    VarAttrs[('longitude', 'MetaData')]['units'] = 'degree'
-    VarAttrs[('latitude', 'MetaData')]['units'] = 'degree'
+    VarAttrs[('waterMeridionalCurrentSpeed', 'ObsValue')]['units'] = 'm s-1'
```

Comment:
We should be able to re-use the variable names in list higher up:  ```vname[0]``` and ```vname[1]``` so there is only one spot in the code to change.